src/tvpack.jl

```diff
@@ -124,10 +124,10 @@ end
 # # One Dimensional Globally Adaptive Integration Function
 
 function adonet(f::Function, a::Float64, b::Float64, tol::Float64)
-	ai = Array(Float64, 100)
-	bi = Array(Float64, 100)
-	ei = Array(Float64, 100)
-	fi = Array(Float64, 100)
+	ai = Array{Float64}(100)
+	bi = Array{Float64}(100)
+	ei = Array{Float64}(100)
+	fi = Array{Float64}(100)
```

Comment:
Might make it easier on the compiler to use `Vector` here so it doesn't have to determine the number of dimensions itself.